src/types/JobListing.ts

```diff
@@ -0,0 +1,12 @@
+type JobListing = {
+    title:string,
+    company: string,
+    logoUrl: string,
+    location: string,
+    jobPostingUrl: string,
+    salary: string,
+    publicationDate: string,
```

Comment:
Might be helpful to add a comment here that we expect the format to be YYYY/MM/DD